tests/Database/Functional/Driver/MySQL/Schema/ConsistencyTest.php

```diff
@@ -31,4 +31,17 @@ public function testSet(): void
         $this->assertSame(['a', 'b', 'value'], $column->getEnumValues());
         $this->assertSame(['a', 'b', 'value'], $schema->column('target')->getEnumValues());
     }
+
+    public function testSmallPrimary(): void
+    {
+        $schema = $this->schema('table');
+        $this->assertFalse($schema->exists());
+
+        $column = $schema->smallInteger('target');
+
+        $schema->save();
+        $schema = $this->schema('table');
+        $this->assertTrue($schema->exists());
+        $this->assertTrue($schema->column('target')->compare($column));
```

Comment:
Please add check that
`$column` is not the same object as `schema->column('target')`